include/tcpdf_cmssignature.php

```diff
@@ -0,0 +1,545 @@
+<?php
+/*
+// File name   : tcpdf_cmssignature.php
+// Version     : 1.0.1
+// Begin       : 2023-05-25
+// Last Update : 2023-05-25
+// Author      : Hida - https://github.com/hidasw
+// License     : GNU GPLv3
+*/
+/**
+ * @class tcpdf_cms_signature
+ * Manage CMS Signature for TCPDF.
+ * @version 1.0.001
+ * @author M Hida
+ */
+class tcpdf_cms_signature {
+	/**
+	 * Catch error.
+	 * @public
+	 */
+	public string $errorMsg;
```

Comment:
```suggestion
	public $errorMsg;
```

You need to remove them, old php compatibility is required

Reply:
Glad to see you actively maintain this repo